odoo/addons/base/tests/test_ir_cron.py

```diff
@@ -205,42 +191,28 @@ def test_cron_keep_future_triggers(self):
         self.assertTrue(capture.records, "trigger should has been kept")
 
     def test_cron_process_job(self):
```

Comment:
also noting it here because I cannot add a review for `test_cron_ready_jobs_order`. Now that you changed the `ORDER BY` inside the `_get_all_ready_jobs` method. We should the add a few test cases inside/next to `test_cron_ready_jobs_order`